mapit_labour/management/commands/mapit_labour_import_addressbase_core.py

```diff
@@ -104,4 +104,7 @@ def create_uprn(self, row: Dict[str, str]):
 
     def print_stats(self):
         c = self.count
-        print(f"Imported {c['total']} ({c['created']} new, {c['updated']} updated)")
+        print(
+            f"Imported {c['total']} ({c['created']} new, {c['updated']} updated)",
+            file=self.stdout,
+        )
```

Comment:
https://docs.djangoproject.com/en/3.2/howto/custom-management-commands/#management-commands-output recommends self.stdout.write (I assume it's the same, though).